dotcom-rendering/src/web/components/ArticleBody.tsx

```diff
@@ -104,13 +104,10 @@ export const ArticleBody = ({
 	) {
 		return (
 			<>
-				{format.design === ArticleDesign.LiveBlog && (
-					<span data-gu-marker="top-of-blog" />
-				)}
 				<div
 					// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
 					tabIndex={0}
-					id="maincontent"
+					id="liveblog-body"
```

Comment:
Would this affect other things that rely on the "maincontent" id? E.g. accessibility skip-to links?

Reply:
I've moved `#maincontent` to higher up (it now sits just above key events) so things like skip to links and general page navigation will still work. 

There's a change here though, previously we navigated users tight to the top of the blocks but now they arrive further up, and can see additional items.

### Before
<img width="1004" alt="Screenshot 2022-03-01 at 11 25 17" src="https://user-images.githubusercontent.com/1336821/156160850-0de8f197-1e7d-4beb-b688-bbebfcb1a022.png">


### After
<img width="1004" alt="Screenshot 2022-03-01 at 11 25 03" src="https://user-images.githubusercontent.com/1336821/156160856-345f65ec-3804-43bc-b3fb-917c56dc78f8.png">


What do you think? In my opinion, including this extra content at the top of the page when readers navigate is an improvement but it's perhaps not _semantically_ the actual content.

Reply:
tl;dr I think this is better

I always found it confusing when I got scrolled to the top of the first post because I wasn't sure if I was at the top or in the middle, so always ended up scrolling a bit further up to get my bearings. When we designed the filter key events button UX also wanted us to include the button when scrolling to the top (perhaps a better arguments than my subjective world view).

Reply:
> I always found it confusing when I got scrolled to the top of the first post because I wasn't sure if I was at the top or in the middle, so always ended up scrolling a bit further up to get my bearings

Me too!

> When we designed the filter key events button UX also wanted us to include the button when scrolling to the top

Perfect. This is good to hear